src/syft/lib/python/complex.py

```diff
@@ -1,25 +1,203 @@
 # stdlib
 from typing import Any
+from typing import List
 from typing import Optional
 
+# third party
+from google.protobuf.reflection import GeneratedProtocolMessageType
+
 # syft relative
+from ... import deserialize
+from ... import serialize
 from ...core.common import UID
+from ...core.store.storeable_object import StorableObject
 from ...decorators import syft_decorator
+from ...proto.lib.python.complex_pb2 import Complex as Complex_PB
+from ...util import aggressive_set_attr
+from .primitive_factory import PrimitiveFactory
 from .primitive_interface import PyPrimitive
 
 
 # TODO - actually make all of this work
 class Complex(complex, PyPrimitive):
     @syft_decorator(typechecking=True, prohibit_args=False)
-    def __new__(self, value: Any = None, id: Optional[UID] = None) -> "Complex":
-        if value is None:
-            value = 0.0
-        return complex.__new__(self, value)  # type: ignore
+    def __new__(
+        self, real: Any = None, imag: Any = None, id: Optional[UID] = None
+    ) -> "Complex":
+        if real is None:
+            return complex.__new__(self)
+        if imag is None:
+            return complex.__new__(self, real=real)  # type: ignore
+        if isinstance(real, str):
```

Comment:
Q: Can't we add this to real: ```Union[int, str]```?

Reply:
I think it can also be a float but I can add this types!

Reply:
Finally, real can be of any type. In the tests, they try to instantiate a complex with a custom class and expect that to work: `self.assertEqual(Complex(OS(1 + 10j)), 1 + 10j)`